plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -46,12 +46,10 @@ import {
 } from '@opentelemetry/semantic-conventions';
 import { VERSION } from './version';
 
-const PG_POOL_COMPONENT = 'pg-pool';
-
 export class PgInstrumentation extends InstrumentationBase {
   static readonly COMPONENT = 'pg';
 
-  static readonly BASE_SPAN_NAME = PgInstrumentation.COMPONENT + '.query';
+  static readonly BASE_SPAN_NAME = 'query';
```

Comment:
As this variable is only used in `utils.ts`, and now only contains the value 'query' without any component part, I suggest that we move it to `utils.ts` as const, and give it a better name, like dbOperationQuery which communicate it's role more clearly